src/App.vue

```
@@ -1,183 +1,54 @@
-                <v-list-item-title>Profile</v-list-item-title>
-              </v-list-item>
-              <v-list-item link @click="auth.logout()">
-                <v-list-item-title>Logout</v-list-item-title>
-              </v-list-item>
-            </v-list>
-          </v-menu>
-        </div>
+  <main>
+    <h1 class="sr-only">Lista de Vagas</h1>
+    <section v-if="vacancies.length" aria-label="Lista de vagas disponíveis">
+      <div v-for="vacancy in vacancies" :key="vacancies.id" class="vacancies-card">
```

ESLint

[error] Expected 'v-bind:key' directive to use the variables which are defined by the 'v-for' directive.

(vue/valid-v-for)